### PR TITLE
docs: write initial documentation

### PR DESCRIPTION
**Pull request by pevisscher**

### CODE REVIEW

**Comment by pevisscher** [comment]
Current dependencies on/for this PR:
* main
  * **PR #61** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/61" target="_blank">Graphite</a> 
    * **PR #62** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/62" target="_blank">Graphite</a> 
      * **PR #63** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/63" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/therefore/63?utm_source=stack-comment).

File: README.md [hunk 1]
```diff
@@ -12,39 +12,25 @@
 
 > Such is the advantage of a well-constructed language that its simplified notation often becomes the source of profound theories.
 >
-> -   Pierre-Simon de Laplace
-Therefore empowers you to generate JSON Schemas and typescript types.
+> -   Pierre-Simon de Laplace Therefore empowers you to generate JSON Schemas and typescript types.
```
Comment (MickVanDuijn) [inline README.md:15]:
```suggestion
> -   Pierre-Simon de Laplace

Therefore empowers you to generate JSON Schemas and typescript types.
```

File: docs/docs/intro.md [hunk 2]
```diff
@@ -1,9 +1,33 @@
...
 title: Overview
 slug: /
 ---
 
 # Introduction
 
-This is the package documentation
+> Such is the advantage of a well-constructed language that its simplified notation often becomes the source of profound theories.
+>
+> -   Pierre-Simon de Laplace Therefore empowers you to generate JSON Schemas and typescript types.
```
Comment (MickVanDuijn) [inline docs/docs/intro.md:11]:
```suggestion
> -   Pierre-Simon de Laplace

Therefore empowers you to generate JSON Schemas and typescript types.
```

**Review by MickVanDuijn — APPROVED** [review]

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/therefore/63?utm_source=stack-comment).


Job Reference: JzO6cwGTfENaZFy5T9Os

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/therefore/63?utm_source=stack-comment).


Job Reference: ndSaF8BbFfaLQTchrjOV

**Pull request merged**